framework/src/main/java/com/github/dbmdz/flusswerk/framework/reporting/DefaultProcessReport.java

```diff
@@ -79,7 +81,7 @@ public void reportComplexRetry(Message message, RetryProcessingException e) {
             messagesSent,
             e.getMessage(),
             keyValue("amqp_message", message.toString()),
-            keyValue("exception", e.toString()));
```

Comment:
Why is this line deleted? It is still included in `reportFail` for example.

Reply:
Should have been deleted there as well, not sure what happened. This replicates parts of the exception logging functionality from logback and nis not needed when we log e as instance of Exception.